include-refactoring/xsimd/types/xsimd_api.hpp

```diff
@@ -1528,8 +1536,8 @@ batch<T, A> zip_lo(batch<T, A> const& x, batch<T, A> const& y) {
  * @return a boolean scalar.
  */
 template<class T, class A>
-bool all(batch_bool<T, A> const& x) {
-  return kernel::all<A>(x, A{});
+bool all(batch_bool<T, A> const& self) {
```

Comment:
please keep `x` so that it matches the doxygen

Reply:
Will fix that